demisto_sdk/commands/validate/validators/MR_validators/MR101_validate_empty_keys.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.content_graph.objects.modeling_rule import ModelingRule
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = ModelingRule
+
+
+class ValidateEmptyKeysValidator(BaseValidator[ContentTypes]):
+    error_code = "MR101"
+    description = (
+        "Validate that the modeling rules keys - rules and schema exist and are empty"
+    )
+    rationale = "This standardization is to handle modeling rules correctly."
+    error_message = (
+        "Either the 'rules' key or the 'schema' key are missing or not empty, "
+        "make sure to set the values of these keys to an empty string."
+    )
+    related_field = "modeling rule"
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        return [
+            ValidationResult(
+                validator=self,
+                message=self.error_message,
+                content_object=content_item,
+            )
+            for content_item in content_items
+            if content_item.schema_key != "" or content_item.rules_key != ""
```

Comment:
```suggestion
            if content_item.schema != "" or content_item.rules != ""
```

Reply:
This should work due to the alias

Reply:
This is not correct. 
In the modeling rules, I added it as a schema_key and rules_key. And so is the property. 
I can't use `schema` because Pydetic already uses it

Reply:
The alias lets you get to the field itself, that's why it works.
But it does not know what rules are as part of the object